pontoon/base/migrations/0009_change_pontoon_users_emails.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 2.2.13 on 2020-08-28 08:49
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def update_system_user_email(apps, schema_editor):
+    User = apps.get_model("auth", "User")
+    User.objects.filter(email="pontoon-sync@mozilla.com").update(
+        email="pontoon-sync@example.com"
+    )
+    User.objects.filter(email="pontoon-gt@mozilla.com").update(
+        email="pontoon-gt@example.com"
+    )
+    User.objects.filter(email="pontoon-mt@mozilla.com").update(
```

Comment:
pontoon-tm@mozilla.com

Reply:
I am Really Sorry for all the typos. Won't happen again